src/basic_block.rs

```diff
@@ -146,6 +146,7 @@ impl BasicBlock {
     }
 
     /// Prepends one `BasicBlock` before another.
+    /// It returns `Err(())` when either `BasicBlock` has no parent, as LLVM assumes it has a parent.
```

Comment:
"as LLVM assumes it has a parent." --> "as LLVM assumed they both have parents."